text/intro.md

```diff
@@ -1,8 +1,6 @@
-#Introduction
+#Introduction to Text-Analysis
```

Comment:
Why is this `Introduction to Text-Analysis`? The left nav says `Text analysis` (no dash, no "Introduction to", no capital letter on analysis). I think they should match.